charts/vault-monitoring/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: vault-monitoring
 description: monitor your vault server from within Kubernetes' prometheus
 type: application
-version: 0.1.4
-home: https://github.com/adfinis-sygroup/helm-charts/tree/main/charts/vault-monitoring
+version: 0.2.0
+home: https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/vault-monitoring
```

it looks like automerge picked the wrong side of the merge for this line:
```suggestion
home: https://github.com/adfinis-sygroup/helm-charts/tree/main/charts/vault-monitoring
```